slave.mk

```diff
@@ -1029,6 +1029,24 @@ $(foreach IMAGE,$(DOCKER_IMAGES), $(eval $(IMAGE)_FILES_PATH := $(FILES_PATH)))
 $(foreach IMAGE,$(DOCKER_DBG_IMAGES), $(eval $(IMAGE)_DEBS_PATH := $(DEBS_PATH)))
 $(foreach IMAGE,$(DOCKER_DBG_IMAGES), $(eval $(IMAGE)_FILES_PATH := $(FILES_PATH)))
 
+# Targets for downloaded docker images
+$(addprefix $(TARGET_PATH)/,$(DOWNLOADED_DOCKER_IMAGES)) : $(TARGET_PATH)/%.gz : .platform \
+		$$(%.gz_DEP_FILES)
+	$(HEADER)
+
+	cp files/$(DOWNLOADED_DOCKER_IMAGES) target/$(DOWNLOADED_DOCKER_IMAGES)
```

Comment:
It is not downloaded from URL?

Reply:
Did you build the image locally?

Reply:
@shanshri and @lguohan and @ashwin-h - @liushilongbuaa mentioned that we need AMD to provide a url to download the AMD/Pensando package in this PR.  SONiC team needs to get it (by URL) when building.

Reply:
@liushilongbuaa - the AMD team can share the file/private URL with you. How do they do this?

Reply:
Hi @lguohan , If the file is private, can we publish the SONiC image? Can we publish target/docker-dpu.gz?

Reply:
@shanshri , do the docker-base URL have secrets?
If you don't provide the URL. We can't build image for pensando.
Can you provide a general URL without secrets? Thus we can go through the build pipeline.

Reply:
It is azure's resource. Only people who are microsoft FTE and SONiC team can upload.

Reply:
Do you have an example that i can follow? I am planning to upload a dummy dpu docker so that sonic compilation goes through.

Reply:
https://sonicstorage.blob.core.windows.net/public/docker-dpu-base.gz
Here is a fake docker image. It is ubuntu image. I tagged it with docker-dpu-base:latest.

Reply:
Done